### DISCUSSION TITLE
Voltronic Axpert family

### DISCUSSION THREAD

**Discussion by noossyy** [Sleeper85/esphome-jk-bms-can#22] [Q&A]
Hello,
I have a Voltronic Axpert Max 2 8kw, it is advertised as compatible with Pylontech protocol:

![IMG_3812](https://github.com/Sleeper85/esphome-jk-bms-can/assets/101205659/75b9392e-9b5f-48a1-9500-e65f13a5410a)
![IMG_3813](https://github.com/Sleeper85/esphome-jk-bms-can/assets/101205659/f39b5c49-fc55-426b-9a86-a6983b7fc987)
![IMG_3814](https://github.com/Sleeper85/esphome-jk-bms-can/assets/101205659/d5b1d499-336b-48e6-86fa-b58556322b98)

It has a BMS rj45 port which is supposed to handle RS485 and CAN .
I was wondering if someone has ever tried already?

**Comment by Sleeper85** [comment]
Hi,

Voltronic inverters are not supported at this time. But from the links below it appears to be able to communicate via CAN bus with a Weco or Soltaro battery with totally different CAN IDs than what is used in the current application. It would therefore be necessary to add a new protocol dedicated to Voltronic.

https://forums.aeva.asn.au/viewtopic.php?p=88506#p88506

**Reply by noossyy** [reply]
Thanks for reply and bravo for your hard work maintaining this beautiful project. 
Regarding compatibility, if you say that Axperts are not supported, then what is this PYL option in the settings of the inverter? 
just for my personal knowledge.

**Comment by Sleeper85** [comment]
PYL is for a Pylontech battery with RS485 communication (not CAN BUS).

The simplest would be to add the CAN bus protocol from Weco or Soltaro but I couldn't find any official document.
Only the links I posted above.
